cluster-api-gcp/.gitignore

```diff
@@ -1,2 +1,3 @@
 cluster-api-gcp
 machine-controller/machine-controller
+machines.yaml
```

Comment:
Have you also deleted machines.yaml from the repo?

Reply:
Look at the commit below. I moved machines.yaml to machines.yaml.template.
https://github.com/kubernetes/kube-deploy/pull/555/commits/1626a31e74d25420d4241cd4f23986b4c2b7acf2